src/main/java/fi/helsinki/cs/tmc/core/TmcCore.java

```diff
@@ -203,7 +162,7 @@ public Path getExerciseChecksumCacheLocation() {
      * @param path inside any exercise directory
      * @return SubmissionResult object containing details of the tests run on server
      * @throws TmcCoreException if there was no course in the given path, no exercise in the given
-     * path, or not logged in
+     path, or not logged in.
```

Comment:
indentation is incorrect:  When an at-clause doesn't fit on a single line, continuation lines are indented four (or more) spaces from the position of the @.
https://google.github.io/styleguide/javaguide.html#s7.1.3-javadoc-at-clauses